index.html

```diff
@@ -4116,91 +4076,298 @@ <h3>Parsing content</h3>
+          minimize user data exposed to NFC.
+        </td>
+      </tr>
+      <tr>
+        <td>Malicious web page overwrites tags without user consent.</td>
+        <td>
+          Make sure the write is additive to previous content.
+          Or, require permission and user prompt needed for writing tags.
+          Or, control what tags can be written by a given web page, for instance
+          a web page can write only a tag which can be connected to its origin.
+          Or, allow overwriting since tags not meant to be written can be
+          protected by making them read only.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Malicious web page writes sensitive user data to tags and peer
+          devices.
+        </td>
+        <td>
+          The user should be able to be aware of what data can be shared using
+          NFC from the given web page.
+          Use permission/user prompt for writing tags and peers.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Malicious tag may be involuntarily or voluntarily read by devices
+          and the data read may constitute an attack vector on the user agent.
+        </td>
+        <td>
+          This is a generic problem with all existing NFC tags.
+          The data is considered application specific.
+          Implementations need security hardening.
+          Involuntary touch is low probability due to short range and critical
+          angle for reading.
+          It’s easier to attack elsewhere (e.g. WiFi and Bluetooth).
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Malicious smart poster tag attempts triggering an action on the
+          device which may be a threat.
+        </td>
+        <td>
+          Support for smart posters is not among the main use cases of Web
+          NFC; if and when supported, no automatic actions should be allowed,
+          the user must be made aware and given the ability to control what is
+          happening during the NFC communication.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Malicious Web NFC tag could send an arbitrary payload to a website
+          via a user’s device, along with a user’s existing web credentials.
+          The webapp and server would need to treat that payload as completely
+          untrusted.
+        </td>
+        <td>
+          Suggest rules for handling payload safely, provide best-practice
+          methods for doing so, provide a sanitization/validation function.
+          Payload MAY be even cryptographically signed before writing it to a
+          tag so the contents could later be verified.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          A Web NFC tag could be used for leaking the user’s physical location,
+          <ul>
+            <li>
+              if the reading triggers a user’s device to navigate to a website
+              that can identify the tag and knows the tag's physical location.
```

Comment:
(if they know where that tag is location, ie serial number or encoded data)

Reply:
Try again that sentence, I don't get the point.

Reply:
You don't explain how they can identify the tag and use that to detect your physical location.

They can because tags can carry serial numbers or unique data that can be associated with the physical placement of the tag that was read

Reply:
This is a short formulation of _what_ the threat is; it would be besides the point to maintain _how_ a website can identify the tag - what is important is if it _can_ identify the tag and knows where the tag was left off (again, we don't need to explain in what ways can you maintain a tag's physical location, as there are quite many options).